Complete check-in #2 questionnaire

ryanflach opened the pull request:
Add feedback for second check-in. 

@allindow Please review =) Left the section on the stretch goal as pending - we can adjust that section and next steps based on the initial progress you're able to make on collision detection this evening.

allindow commented on 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md:
```diff
@@ -0,0 +1,80 @@
+  - Did you deviate from the instructions? Tell us about it.
+    * No, we stuck to what was originally discussed.
+  - Questions, Comments, Concerns?
+    * We still have some concerns regarding performance, but these issues are not consistent.
```
@ryanflach do you mean the choppiness of the background?

ryanflach replied:
@allindow Yes - were you experiencing this as well? I didn't notice it this morning.

allindow commented on 1605/module_4_assignments/gametime/rf_al_game_time/ryan_angela_check_in_2.md:
```diff
@@ -0,0 +1,80 @@
+## Next Steps?
+
+- What do you feel the next steps should be?
+  * Implement game start and game end functionality
```
How about a...not sure what you would call it? Welcome screen giving a quick definition of a jeepney and showing the different obstacles/bonuses with what they're called. Or would that come after clicking start?

ryanflach replied:
I like that idea. Our game instructions will be very simple, so we should be able to add some background information on the screen where they would click start.

ryanflach commented:
@allindow I've updated and added to the stretch goal based on your completion of that feature. Want to give it a final look over before I remove the [WIP] tag from the PR?

allindow commented:
This looks good
